build-package.bash

```diff
@@ -13,10 +13,33 @@ rm -rf packagebuild
 mkdir -p packagebuild/debian 
 
 git log -n 1 --pretty=format:%h.%ai.%s > commitstring.txt
-export DEBFULLNAME=`git log -n 1 --pretty=format:%an`
-export DEBEMAIL=`git log -n 1 --pretty=format:%ae`
-export DEBBRANCH=`echo "${BRANCH}" | sed 's/[\/\_]/-/g'`  
-export DEBPKGVER=`git log -n 1 --pretty=oneline --abbrev-commit`
+if [ -z "$DEBFULLNAME" ]; then
+	export DEBFULLNAME=`git log -n 1 --pretty=format:%an`
+fi
+
+if [ -z "$DEBEMAIL" ]; then
+	export DEBEMAIL=`git log -n 1 --pretty=format:%ae`
+fi
+
+if [ -z "$DEBBRANCH" ]; then
+	export DEBBRANCH=`echo "${BRANCH}" | sed 's/[\/\_]/-/g'`  
+fi
+
+if [ -z "$DEBPKGVER" ]; then
+  export DEBPKGVER=`git log -n 1 --pretty=oneline --abbrev-commit`
+fi
+
+if [ -z "$DCHOPTS" ]; then
+	export DCHOPTS="-l ${DEBBRANCH} -u low ${DEBPKGVER}"
+fi
+
+echo "DEBDIR:       $DEBDIR"
+echo "DEBFULLNAME:  $DEBFULLNAME"
+echo "DEBEMAIL:     $DEBEMAIL"
+echo "DEBBRANCH:    $DEBBRANCH"
+echo "DEBPKGVER:    $DEBPKGVER"
+echo "DCHOPTS:		  $DCHOPTS"
```

Comment:
a little obsessive-compulsive personality, this line is not well-aligned with above :)